### DISCUSSION TITLE
Unlocks after Scenario 4 - Spoilers

### DISCUSSION THREAD

**Discussion by gavinctaylor** [Lurkars/gloomhavensecretariat#187] [Q&A] [answered]
Hi, 
So far the scenario and campaign tracking in GHS has been brilliant. Thank you for the hard work. I have a question about scenario unlocks from scenario 4 and didn't know if it was a bug or somthing I am missing.
I completed scenario 4 (starting by taking the B option). When I ended I was prompted to read 15.2. This allowed me to unlock the Fist class (yay) as well as Scenario 6 - Avalanche. 
I was prompted to read 63.1, which unlocks Scenario 7 (Edge of the World), and Scenario 8 (Crystal Trench). I did not see anywhere instructions to unlcok scenario 5.

When marking Scenario 4B on the campaign sheet of GHS all 4 scenarios are now available (5, 6, 7, 8). 

From my reading I don't think Scenraio 5 should be available. As I said I don't know if this is a bug, or if Scenario 5 really is available and I missed something in the text.

**Comment by Lurkars** [comment, marked as answer]
Thanks, you're right. I just used scenario flow chart, but it's clear stated that book > chart, and in book you only unlock 5 or 6 depending on ending. I will fix that!

**Reply by gavinctaylor** [reply]
Brilliant! Thank you so much for fixing.
